icinga2/resource_icinga2_host.go

```diff
@@ -30,6 +30,14 @@ func resourceIcinga2Host() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"groups": &schema.Schema{
+				Type:     schema.TypeList,
+				Required: true,
```

Comment:
With release v0.2.1 you should be able to make groups optional now. So this can be merged in and not break backwards compatability